### PR TITLE
Implement GetUser service

### PR DESCRIPTION
**Pull request by kimlisa**
- golinted and formatted
- added getUserRow db utility function
- unit tested GetUser

screen shot 2019-01-20 at 4 31 35 am
screen shot 2019-01-20 at 4 31 46 am

### CODE REVIEW

File: service/db.go [hunk 1]
```diff
@@ -232,3 +234,40 @@ func deleteUser(uuid string) error {
 
 	return nil
 }
+
+func getUserRow(uuid string) (*pb.User, error) {
```
Comment (kimlisa) [inline service/db.go]: This service is very identical to `DeleteUser` Service.
The following lines of code is similar except for different error message:

```	
        if req == nil {
		return nil, status.Error(codes.InvalidArgument, errNilRequestUser.Error())
	}

	if err := refreshDBConnection(); err != nil {
		return nil, status.Error(codes.Unknown, err.Error())
	}

	// get User Object
	user := req.GetUser()
	if user == nil {
		logger.Error(errNilRequestUser.Error())
		return nil, status.Error(codes.InvalidArgument, errNilRequestUser.Error())
	}

	// validate user id
	uuid := user.GetUuid()
	if err := validateUUID(uuid); err != nil {
		logger.Error("GetUser validating uuid: ", err.Error())
		return nil, status.Error(codes.InvalidArgument, err.Error())
	}

	// check uuid exists
	exists, err := checkUserExists(uuid)
	if err != nil {
		logger.Error("GetUser checkUserExists:", err.Error())
		return nil, status.Error(codes.Internal, err.Error())
	}

	if !exists {
		logger.Error(errDoesNotExistUUID.Error())
		return nil, status.Error(codes.NotFound, errDoesNotExistUUID.Error())
	}
```

I am also thinking for UpdateUser, I would also need these exact lines of code before updating. Should I put this into its own utility function?

File: service/db.go [hunk 2]
```diff
@@ -96,7 +96,7 @@ CREATE TABLE user_svc.accounts
   email             VARCHAR(320) NOT NULL UNIQUE,
   password          VARCHAR(60) NOT NULL,
   organization      TEXT,
-  created_date      TIMESTAMP NOT NULL,
+  created_date      TIMESTAMPTZ NOT NULL,
```
Comment (faraonc) [inline service/db.go]: Please update the one in `hwsc-dev-ops` as well

File: service/db.go [hunk 3]
```diff
@@ -232,3 +234,40 @@ func deleteUser(uuid string) error {
 
 	return nil
 }
+
```
Comment (faraonc) [inline service/db.go]: needs documentation

File: service/db.go (hunk 1 above)
Comment (faraonc) [reply service/db.go]: I think we can get away of not checking if the user exists for updating user

**Pull request merged**